src/styles/index.scss

```
@@ -1,3 +1,77 @@
+@mixin styles($color, $height, $width) {
```

```suggestion
@mixin hand-styles($color, $height, $width) {
```
good idea, but let's rename it